src/AnonymizationStep/AnonymizedResultsTable.tsx

```diff
@@ -76,14 +64,23 @@ function renderValue(v: Value) {
   }
 }
 
-function buildCellRenderer(column: AnonymizedResultColumn, bucketColumns: BucketColumn[]) {
+function renderBucketValue(column: AnonymizedResultColumn, bucketColumns: BucketColumn[]) {
```

Comment:
This actually builds the renderer instead of rendering the value. Previous name was more honest.

Reply:
I know, but it just didn't fit with the rest in my opinion. Every time I used to read this, I stopped to think, where it really belongs (despite I wrote the code and named the function originally).

As it is now, it communicates clearly, that it is a sibling of the `renderXYZ` functions, it's just parametrized

Reply:
Ah so you mean F# style currying? Haven't been used to that in js and thought you made a mistake.

Reply:
We have 2 competing terms for "expressions by which we have grouped by". A search for `[Gg]roupingLabel` returns 18 hits, `[Bb]ucketLabel` 3 hits, and `[Bb]ucketValue` 0 hits. Any thoughts?

Reply:
Looks like the discrepancy exists between frontend and anonymizer. If we stick to that, The proper name for this should be `renderBucketColumn`.

Reply:
> Ah so you mean F# style currying?

heh, I wasn't trying to make it that smart, only to appear similar to the other 3, but there's something to it.

> Looks like the discrepancy exists between frontend and anonymizer. If we stick to that...

yup, so this function already talks about `bucketColumns` so I thought `bucketValue` fits in nicely. 

> The proper name for this should be renderBucketColumn.

um, hold on :), we're rendering just one value in a column, not the entire column, maybe you've meant `renderBucketColumnValue` instead (which works for me)? I took `bucketValue` out of the paper BTW, but that's yet another story.

Reply:
> we're rendering just one value in a column, not the entire column

By "column" we don't really know if we're referring to the ant table column or to the SQL column (grouping label)... I meant the latter.

`bucketColumnValue` sounds good because it makes it clearer. I'm still a little unhappy that we have diverged in terminology, but this isn't the place to fix it.